themes/default/content/docs/guides/self-hosted/ecs-hosted.md

```diff
@@ -50,13 +50,13 @@ To this end, you need to set up the following:
 
 ### Deployment Steps
 
-See the [README](https://github.com/pulumi/pulumi-self-hosted-installers/ecs-hosted/README.md) file provided with the installer package for detailed deployment steps.
+See the [README](https://github.com/pulumi/pulumi-self-hosted-installers/tree/master/ecs-hosted/README.md) file provided with the installer package for detailed deployment steps.
```

Comment:
This one's 404ing for me -- looks like the README is lowercase.

We should probably rename it in the repo, though. What do you think @MitchellGerdisch?